code/Modules/Roles/RoleDistributor.cs

```diff
@@ -12,7 +12,8 @@ public class RoleDistributor
     {
         private HashSet<MainPlayer> players;
         public Scenario Scenario;
-
+        public Role DefaultRole = new Assistant();
```

Comment:
Default should be in Scenario

Reply:
Yes but I don't want to change how the scenario is created for now. I put this here for reduce the dept. If you really want i can make the change

Reply:
Nope it's fine but we'll change it in the future maybe